### PR TITLE
Fix index data no update #390

### PR DESCRIPTION
**Pull request by clang-clang-clang**

### CODE REVIEW

**Review by Shchvova — COMMENTED** [review]
fixing indents

**Comment by Shchvova** [comment]
Not a big deal, but minor issue. Is it possible to give me push permission to push to PR or change spaces to tabs. It is silly and won't block the merge.

**Review by Shchvova — APPROVED** [review]

**Comment by clang-clang-clang** [comment]
> Looks nice. Do you want "DIANCHU TECHNOLOGY" in all caps?

I am discussing this and maybe we can change it next time.

**Pull request merged**
